game/songs.cc

```diff
@@ -233,16 +213,18 @@ void Songs::CacheSonglist() {
 	}
 
 	fs::path cacheDir = getCacheDir() / SONGS_CACHE_JSON_FILE;
-
+	std::ofstream outFile;
 	try {
-		std::ofstream outFile(cacheDir.string());
```

Comment:
why not creating the stream in the try catch block? you could use the constructor instead of open

Reply:
Because then I can't close it outside of the try block.

Reply:
as the destructor closes the file, you don't actually need to close it ouside the try block